docs/docs/glossary.md

```diff
@@ -192,6 +196,10 @@ JSX is an extension to JavaScript that allows developers to write HTML and custo
 
 Linting is the process of running a program that will analyze code for potential errors. The Gatsby project uses [prettier](https://prettier.io/) to identify and fix common style issues. Another example of a linter commonly used in React projects is [eslint-jsx-plugin-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y), which checks for common [accessibility](#accessibility) issues in development.
 
+### LocalStorage
+
+LocalStorage is the storage that to save key/value pairs in a web browser. The localStorage object stores data with no expiration date. The data will not be deleted when the browser is closed, and will be available the next day, week, or year.
```

Comment:
Same deal as with MDN. This is a little too similar to the definition on w3schools.